.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -575,14 +575,14 @@ stages:
       displayName: Download MSI
       inputs:
         artifact: windows-msi-$(targetPlatform)
-        patterns: '**/*profiler-beta.msi'
+        patterns: '**/*-x64.msi'
```

Comment:
I think this should be `/**/*-$(targetPlatform).msi` or `/**/*.msi` because there should only be one MSI in the `windows-msi-x64`/`windows-msi-x86` artifact folder now, and it ends with the same architecture suffix as the folder